src/aind_ephys_rig_qc/qc_figures.py

```diff
@@ -56,16 +72,142 @@ def plot_power_spectrum(data, sample_rate, stream_name):
     """
 
     fig = Figure(figsize=(10, 4))
-    ax = fig.subplots()
+    ax = fig.subplots(2, 1, gridspec_kw={"height_ratios": [2, 1]})
 
     subset = data[:1000, :]
```

Comment:
Would it make sense to sample the recording at multiple points?

If we switch to spikeinterface, we have a conveniente `get_random_data_chunks` function

Reply:
Makes sense! Let me try it.

Reply:
Updated in the new commits.